utils/httputils/utils.go

```diff
@@ -85,35 +85,29 @@ func WriteJSON(w http.ResponseWriter, v interface{}) error {
 	return WriteJSONStatus(w, http.StatusOK, v)
 }
 
-// HandleJSON returns an http.HandleFunc that serves a JSON-encoded data
-// chunk of an object.
-func HandleJSON(v interface{}) http.HandlerFunc {
-	return func(w http.ResponseWriter, req *http.Request) {
-		data, err := json.Marshal(v)
-		if err != nil {
-			WriteError(w, err)
-			return
-		}
-		w.Header().Set("Content-Type", "application/json")
-		_, _ = w.Write(data)
-	}
-}
-
-// HandleJSONData returns an http.HandleFunc that serves a JSON-encoded data
+// HandleStaticJSONData returns an http.HandleFunc that serves a JSON-encoded data
 // chunk.
-func HandleJSONData(data []byte) http.HandlerFunc {
-	return HandleData("application/json", data)
+func HandleStaticJSONData(data []byte) http.HandlerFunc {
+	return HandleStaticData("application/json", data)
 }
 
-// HandleData returns an http.HandleFunc that serves a data chunk with a
+// HandleStaticData returns an http.HandleFunc that serves a data chunk with a
 // specified content-type.
-func HandleData(ct string, data []byte) http.HandlerFunc {
+func HandleStaticData(ct string, data []byte) http.HandlerFunc {
```

Comment:
The `static` part of the method confuses me. The method handles the data passed to it. `static` let's me think that the method does the name interdependently of it's arguments, which is not the case.